editor/editor_settings_dialog.cpp

```diff
@@ -134,8 +134,6 @@ void EditorSettingsDialog::_notification(int p_what) {
 
 		case EditorSettings::NOTIFICATION_EDITOR_SETTINGS_CHANGED: {
 			_update_icons();
-			// Update theme colors.
-			inspector->update_category_list();
```

Comment:
The colors are updated anyways after removing this, so looks like it's ok to remove it. Could someone test?